docs/listeners/api.rst

```diff
@@ -154,6 +154,35 @@ Response format
 
 The default response format for both XML and JSON has two root keys, ``success`` and ``data``. It's possible to add
 your own root keys simply by using ``_serialize`` on the view var.
+For example you would like to add variable ``my_extra_data`` you have to add it to the configuration of the action like, this:
+
+.. code-block:: php
+
+  <?php
+  class MyAction extends BaseAction 
```

Comment:
Instead of an example using a custom action class, an example which uses `$this->Crud->action()->setConfig()` would be more suitable.

Reply:
You mean like, so?

```
class SomeController extends AppController {

public function initialize() {
      $this->Crud->setConfig([
            'serialize' => [
                'my_extra_data'
            ],
            'api' => [
                'success' => [
                    'data' => [
                        'entity' => [],
                        'my_extra_data'
                    ]
                ]
            ]
        ]);
}
```

Reply:
Yes, though you missed the `->action()` in the call chain and it should be done in `beforeFilter()` or controller's action method.

Reply:
Personnaly, I'm using it like this : 

```
$this->set(compact('aggregation'));
$this->Crud->action()->setConfig('serialize.aggregation', 'aggregation');
```